onnxruntime/core/providers/openvino/openvino_execution_provider.cc

```diff
@@ -58,6 +60,14 @@ OpenVINOExecutionProvider::OpenVINOExecutionProvider(const ProviderInfo& info, s
       shared_context_{std::move(shared_context)},
       ep_ctx_handle_{session_context_.openvino_sdk_version, *GetLogger()} {
   InitProviderOrtApi();
+#ifdef _WIN32
+  session_id_ = global_session_counter_.fetch_add(1) + 1;
```

Comment:
Please document impact on FEIL with this change on a small model like MobileNet with this change

Reply:
This feature wont affect the FEIL. This feature is meant for Performance Benchmarking purpose only & doesnt add any significant runtime overhead, while the applications in production wont be using this flow (as this code is never run in normal cases).